float32[3] delta_position
float32[3] delta_orientation
float32 delta_gripper
